components/application-broker/cmd/eventing-service-instance-migration/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM golang:1.12.10-alpine3.10 as builder
```

Comment:
minor: alpine3.10? golang 1.12? is it possible to bump versions?

Reply:
Let me check

Reply:
it is not so important and let's do not change it